diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt

```diff
@@ -36,6 +38,19 @@ inline class RulesConfigYamlTest(private val pathMap: Map<String, String> =
         checkComments("../diktat-analysis.yml")
     }
 
+    @Test
+    fun `check kotlin version`() {
+        val currentKotlinVersion = KotlinVersion.CURRENT
+        pathMap.keys.forEach { path ->
+            val config = readAllRulesFromConfig(path)
```

Comment:
Shouldn't you read from map entry value here, not key?

Reply:
No, i shouldn't